pkg/virt-config/config_test.go

```diff
@@ -256,4 +257,22 @@ var _ = Describe("ConfigMap", func() {
 		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{})
 		Expect(clusterConfig.GetMachineType()).To(testutils.SatisfyAnyRegexp(clusterConfig.GetEmulatedMachines()))
 	})
+
+	table.DescribeTable("SMBIOS values from kubevirt-config", func(value string, result virtconfig.SmbiosConfig) {
+		clusterConfig, _ := testutils.NewFakeClusterConfig(&kubev1.ConfigMap{
+			Data: map[string]string{virtconfig.SmbiosConfigKey: value},
+		})
+		smbios := clusterConfig.GetSMBIOS()
+
+		smbiosJson, err := json.Marshal(smbios)
+		Expect(err).ToNot(HaveOccurred())
+
+		resultJson, err := json.Marshal(result)
+		Expect(err).ToNot(HaveOccurred())
+
+		Expect(string(smbiosJson)).To(BeEquivalentTo(string(resultJson)))
+	},
+		table.Entry("when values set, should equal to result", `{"Family":"test","Product":"test", "Manufacturer":"None"}`, virtconfig.SmbiosConfig{Family: "test", Product: "test", Manufacturer: "None"}),
+		table.Entry("When an invalid smbios value is set, should return default values", `{"invalid":"invalid"}`, virtconfig.SmbiosConfig{Family: "KubeVirt", Product: "None", Manufacturer: "KubeVirt"}),
```

Comment:
When invalid smbios values are set, shouldn't we return an error instead of overwriting by default values?

Reply:
We do throw error [here](https://github.com/kubevirt/kubevirt/pull/2472/files#diff-ff0b8fb5d8669fdd2f69f9351f27d341R293)
But the defaultConfig would set the value in that case
I feel that is okay behavior, but what are your suggestion here? what should we change?